multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,232 @@
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          weETH,
+          [
+            weETH_ONE_JUMP_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[weETH, CHAINLINK_weETH_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [weETH, parseUnits("2", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, 0],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("2", 18)],
+    },
+    {
+      target: VweETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VweETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("2", 18),
+          vTokenReceiver,
+          parseUnits("4600", 18),
+          parseUnits("2300", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WETH, parseUnits("1.9678", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, "0"],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("1.9678", 18)],
+    },
+    {
+      target: VWETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VWETH,
+          parseUnits("0.77", 18),
+          parseUnits("0.8", 18),
```

Comment:
According to the [last recommendations from Chaos labs](https://community.venus.io/t/chaos-labs-launch-parameters-for-liquid-staked-eth-pool-on-arbitrum-06-08-24/4514), CF and LT for WETH on the new pool should be 0

Reply:
Fixed, eb462e035c9d57ad65cdb41577db75d3fc1ee62b